addons/medical/functions/fnc_unconsciousPFH.sqf

```diff
@@ -135,6 +135,19 @@ if (!local _unit) exitwith {
     [(_this select 1)] call cba_fnc_removePerFrameHandler;
 };
 
+if (GVAR(maxUnconsciousTime) > 0 && {!(_unit getvariable [QGVAR(inReviveState), false])} && {(GVAR(level) == 1)}) then {
+    private ["_maxUnitUnconsciousTime"];
+    _maxUnitUnconsciousTime = if ((_unit getvariable [QGVAR(maxDynamicUnconsciousTime), 0]) > 0) then {
```

Comment:
What's this dynamic unconscious time? I don't think that's necessary.

Also the above if statement could be combined with the one on line 146.